Dockerfile

```diff
@@ -1,22 +1,187 @@
+FROM fedora:33 AS builder
+
+# Please keep this list sorted alphabetically
+ENV PACKAGES \
+    bc \
+    brlapi-devel \
+    bzip2 \
+    bzip2-devel \
+    capstone-devel \
+    ccache \
+    clang \
```

Comment:
Do we need both `clang` and `gcc`?

Reply:
Those packages are copied from the upstream fedora dockerfile. The copy has been a no-brainer from that file to avoid loosing time in exploration of installing any dependencies. That said the current config uses cc for building. Nevertheless they are just in the builder stage so they do not bother us at the end of the day.

Reply:
All good.